rs/registry/canister/src/registry.rs

```diff
@@ -772,6 +781,9 @@ mod tests {
 
     #[test]
     fn test_count_fitting_deltas_max_size() {
+        // TODO(NNS1-3746): Make a version of this test where chunking is enabled.
+        let _restore_on_drop = temporarily_disable_chunkifying_large_values();
+
```

Comment:
This test (implicitly) relies on the old behavior, i.e. chunking is disabled. Because chunking is currently disabled, this test would still pass without this chunk. However, we want to be able to enable the feature without reworking a bunch of tests. Similarly, we want to retain the option of quickly disabling the feature, in case something goes wrong. Once the feature has passed its probation period, we can delete this test (assuming we already have a suitable replacement in place, per the TODO).

This same chunk is added to six other tests.

To figure out which tests would break, I simply enabled chunking, ran tests, and observed which ones broke.